sdk/js-query/src/mock/mock.test.ts

```diff
@@ -127,4 +128,108 @@ describe.skip("mocks match testnet", () => {
     // const matchesReal = mock.sign(serializedResponse);
     // expect(matchesReal).toEqual(realResponse.signatures);
   });
+  test("EthCallByTimestampQueryRequest mock matches testnet", async () => {
+    const targetTimestamp =
+      BigInt(Date.now() - 1000 * 30) * // thirty seconds ago
+      BigInt(1000); // milliseconds to microseconds
+    const arbitrumDemoContract = "0x6E36177f26A3C9cD2CE8DDF1b12904fe36deA47F";
+    const data = eth.abi.encodeFunctionSignature("getMyCounter()");
+    const query = new QueryRequest(42, [
+      new PerChainQueryRequest(
+        23,
```

Comment:
Won't this chain stop working very soon? Should you use 10003 (and a different contract address)?

Reply:
Ooh, yes we should update the demo